.github/workflows/generate_document.yml

```
@@ -42,8 +47,16 @@ jobs:
           maturin develop --manifest-path ./crates/voicevox_core_python_api/Cargo.toml --locked
       - name: Generate Sphinx document
         run: sphinx-build docs/apis/python_api public/apis/python_api
+      - name: Generate Javadoc
+        run: |
+          cd crates/voicevox_core_java_api
+          ./gradlew javadoc
+          cd ../..
```

こう書けたりします。あまり一般的ではないかも･･･。
```suggestion
          (cd crates/voicevox_core_java_api && ./gradlew javadoc)
```